lib/imports/api/documents/both/collection.js

```diff
@@ -1,10 +1,10 @@
 import { Mongo } from 'meteor/mongo';
 
 //todo: the collection name and field names should probably be configurable
-export const Documents = new Mongo.Collection('Documents');
+export const documentsColl = new Mongo.Collection('Documents');
```

Comment:
renaming to `documentsColl`. previously in Meteor it was standard to name global objects with a capital, but since now we don't make globals I tend to favor not using the capital and reserving capitals for only class constructors like most other JS projects do. this is just personal preference